### ISSUE TITLE
404 in Interface guidelines > Accessibility tools article

### ISSUE THREAD

**Issue by juliyvchirkov** [primer/figma#6]
hi guys

first of all let me express my appreciations and thankfulness for the hight quality precious stuff you develop and share

exploring [primer.style](https://primer.style/), I've faced the broken link in the article [Interface guidelines > Accessibility tools](https://primer.style/design/accessibility/tools) (repo source: [content/accessibility/tools.mdx](https://github.com/primer/design/blob/master/content/accessibility/tools.mdx])), which points to a missing resource at [figma.com](https://www.figma.com/)

the broken a href encloses the title of the first paragraph of the article (`Accessibility sticker sheet`), the destination [figma.com/file/o3Vhxw4SmBsUjeaVq3nF4y/Accessibility-Notation?node-id=0%3A1](https://www.figma.com/file/o3Vhxw4SmBsUjeaVq3nF4y/Accessibility-Notation?node-id=0%3A1) constantly reports `404` despite of visitor's status (if one logged on [figma.com](https://www.figma.com/) or not) and url escaping (both `?node-id=0%3A1` and `?node-id=0:1` result in `404`), so I can assume the issue on [figma.com](https://www.figma.com/) side is  permanent and requires to be resolved by hands

[inspect the broken element](https://primer.style/design/accessibility/tools#accessibility-sticker-sheet)

<img src=https://i.imgur.com/II5Lf1v.png width="75%">
